src/_includes/_layouts/Home.jsx

```diff
@@ -76,16 +76,19 @@ export default ({ comp, title }) => {
             redirect_url={"about"}
```

Comment:
It might be unrelated to your pr, but can you update the project boxes to be responsive? I see that on the home page, the outline of the box needs to be fixed: 
<img width="464" alt="image" src="https://github.com/user-attachments/assets/453d6583-e227-44f1-bf53-0c4abf0288dd">
This is how it looks in the project page:
<img width="535" alt="image" src="https://github.com/user-attachments/assets/a55f3a25-c923-4a6a-9a05-84aa148bd669">

Reply:
I think this is a padding related issue between the two pages. The card just has a max width of a certain amount of pixels.

Reply:
We should def look into having these be consistent across the website. Another thing I want to keep in mind padding consistency since this will be more noticeable with the changes in #149

Reply:
In #149, I changed the padding of this section to py-16. Just letting you know